tests/conftest.py

```diff
@@ -1,37 +1,32 @@
 """pytest fixtures for moneypenny"""
```

Comment:
Aie, we clearly need to move all this stuff into a library rather than duplicating this much code or having this giant fixtures configuration file. But we can tackle that later.

Reply:
I have a much smaller conftest now.